src/styles/components/data-table.scss

```
@@ -173,4 +173,10 @@
       white-space: nowrap;
     }
   }
+
+  // Tippy link styling
+  .tippy-content a.external-link {
```

```suggestion
  .tippy-content a {
```
Column tooltips will also have internal links to `/help`

---

`a.external-link` is more specific so `a` alone wouldn't override it. Added both.